FirebaseSessions.podspec

```diff
@@ -0,0 +1,45 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseSessions'
+  s.version          = '10.0.0'
+  s.summary          = 'Firebase Sessions'
+
+  s.description      = <<-DESC
+  Firebase Sessions.
```

Comment:
Should have a comment about internal use only.  See https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCoreInternal.podspec#L7

Reply:
Sounds good 👍

Reply:
Yeah re: CocoaPods - I totally did CocoaPods first cause it's what I'm comfortable with. I'll spend some time setting up SPM